### ISSUE TITLE
[UWP][Accessibility] Focusable sibling elements must not have same and localized control type.

### ISSUE THREAD

**Issue by almedina-ms** [microsoft/AdaptiveCards#3932]
# Platform
* UWP

# Version of SDK
master

# Details

Repro Steps:​
1. Launch the application.
2. New screen starts appearing. Navigate to Left pane JSON files.​
3. Now select "ExpenseReport.JSON" button. It will update the Middle and Right Pane. 
4. Navigate to amount section like (Air Travel Expense $300, Auto Mobile Expense $100) etc. at right pane.
5. Verify name of each amount. 

Actual Result:
There is no unique name defined for amount section like(Air Travel Expense $300, Auto Mobile Expense $100) etc. at right pane of Left Pane JSON Files and due to this, screen reader only announce Expand/collapse button which creates confusion for the screen reader user. 

Expected Result:
There should be unique names for sibling controls(Air Travel Expense $300, Auto Mobile Expense $100) etc. that have the same Control type property so that screen reader user could easily identify unique control.
